neon-animated-pages.html

```diff
@@ -67,11 +67,19 @@
       activateEvent: {
         value: ''
       },
-
-      _prevSelected: {
-        type: String
+      
+      /**
+       * If true, the initial page selection will also be animated according to its animation config
+       *
+       * @attribute animate-first-selected
+       * @type boolean
+       * @default false
```

Comment:
I think hydrolysis is able to parse the property block without these annotations.